tests/integration/module/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "private": true,
+  "name": "integration-module-tools",
+  "version": "0.1.0",
+  "jsnext:source": "./src/index.ts",
+  "types": "./dist/types/index.d.ts",
+  "main": "./dist/lib/index.js",
+  "module": "./dist/es/index.js",
+  "scripts": {
+    "test": "jest",
```

Comment:
It seems like module have a standalone jest config, do we need to add the command to CI config?

Reply:
Maybe we should add it. I'm referring to the test:doc, the doc doesn't add a command to CI.